bedrock/settings/base.py

```diff
@@ -679,25 +679,19 @@ def lazy_email_backend():
 STUB_INSTALLER_LOCALES = {
     'release': {
         'win': STUB_INSTALLER_ALL,
-        # should come back in Fx 55
-        # bug 1348610
-        # 'win64': STUB_INSTALLER_ALL,
+        'win64': STUB_INSTALLER_ALL,
     },
     'beta': {
         'win': STUB_INSTALLER_ALL,
-        # should come back in Fx 55
-        # bug 1348610
-        # 'win64': STUB_INSTALLER_ALL,
+        'win64': STUB_INSTALLER_ALL,
     },
     'alpha': {
         'win': STUB_INSTALLER_ALL,
-        # should come back in Fx 55
-        # bug 1348610
-        # 'win64': ['en-us'],
+        'win64': STUB_INSTALLER_ALL,
     },
     'nightly': {
         'win': STUB_INSTALLER_ALL,
-        'win64': ['en-us'],
+        'win64': STUB_INSTALLER_ALL,
```

Comment:
Agreed. I think we can remove the whole thing, but didn't want to do it quite yet in case we need to quickly roll back.